Week1/homework/app.js

```diff
@@ -1,11 +1,163 @@
+ "de_man_van_de_bloemen" : {adress: "https://s.s-bol.com/imgbase0/imagebase3/large/FC/6/7/6/1/9200000077291676.jpg"},
+ "ren_amina_ren" : {adress: "https://s.s-bol.com/imgbase0/imagebase3/large/FC/4/2/9/9/9200000056169924.jpg"},
+ "het_diner" : {adress: "https://www.lezenvooriedereen.be/media/catalog/product/cache/2/image/9df78eab33525d08d6e5fb8d27136e95/d/i/diner_lowres.jpg"},
+ };
+ // step 2.5
 
+ function makeList() {
+  let bookcontainer = document.getElementById("mybooks");
+  let div = document.createElement("div");
+  bookcontainer.appendChild(div);
+  
+
+  for (let i = 0; i<bookTitles.length; i++) {
+    
+    let contentTitle = document.createTextNode(bookDetails[bookTitles[i]].title);
+    let contentLanguage = document.createTextNode(bookDetails[bookTitles[i]].language);
+    let contentAuthor = document.createTextNode(bookDetails[bookTitles[i]].Author);
+    let liTitle = document.createElement("li");
+    let liLanguage = document.createElement("li");
+    let liAuthor = document.createElement("li");
+    let li = document.createElement("li");
+    let content = document.createTextNode(bookTitles[i]);
+    let img = document.createElement("img");
+    
+    /*let detailsTitle = object.values(bookDetails) [i].title;
+    let detailsLanguage = object.values(bookDetails) [i].language;
+    let detailsAuthor = object.values(bookDetails) [i].author;*/
+    
+    liTitle.appendChild(contentTitle);
+    liLanguage.appendChild(contentLanguage);
+    liAuthor.appendChild(contentAuthor);
+    li.appendChild(liTitle);
+    li.appendChild(liLanguage);
+    li.appendChild(liAuthor);
+    li.appendChild(img);
+    img.src = Object.values(bookImg)[i].adress;
```

Comment:
you don't have to use `Object.values` taking the values of `bookImg` you can do so by the `bookTitles[i]` you have, because it's same keys as `bookImg` Right?